### DISCUSSION TITLE
How instancied a `Blob` ?

### DISCUSSION THREAD

**Discussion by BobSquarePants** [brython-dev/brython#2503] [Q&A] [answered]
How can I create a JS Blob object with Bryhton ?

I've tried

```python
Blobby = window.Blob(['test'], {'type':'text/plain'} ).new()
```
but I get
> TypeError: Blob constructor: 'new' is required

Thanks.

**Comment by denis-migdal** [comment, marked as answer]
Isn't it :
```python
Blobby = window.Blob.new(['test'], {'type':'text/plain'})
```

**Reply by BobSquarePants** [reply]
it's look like ! 

it's so counter intuitive , because we give the arguments to new and not to Blob.. but it works :)

**Reply by denis-migdal** [reply]
`Class.new(...args)` is used to say to Brython that we want to do `new Class(...args)`.

Before ES6, all classes were functions, so it was hard to know if `Class()` was a function call or a constructor call.
Nowadays Brython should be able to tell the difference, but I guess this might introduce some breaking changes.
